plots/data_validation.py

```diff
@@ -3,14 +3,18 @@
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
 import os
+import sys
+sys.path.append(os.path.abspath(os.path.join(__file__ ,"../../")))
 import pandas as pd
 import numpy as np
```

Comment:
HI @yerbol-akhmetov, just a minor comment to remove numpy as it is not currently being used.